python/sglang/api.py

```diff
@@ -18,6 +18,7 @@
     SglSelect,
     SglVideo,
 )
+from sglang.srt.server import Runtime as SRTRuntime
```

Comment:
Revert this change. See the comments in L37.
This import will introduce many dependencies (e.g., torch) that we do not want to run on the client side.